neural_lam/train_model.py

```diff
@@ -248,6 +249,18 @@ def main(input_args=None):
     # Load neural-lam configuration and datastore to use
     config, datastore = load_config_and_datastore(config_path=args.config_path)
 
+    # Select model class
+    ModelClass = MODELS[args.model]
+
+    # Prepare graph metadata for graph-based models
+    graph_sizes = None
+    graph_name = args.graph if issubclass(ModelClass, BaseGraphModel) else None
+    if graph_name is not None:
+        graph_dir_path = datastore.root_path / "graph" / graph_name
+        _, graph_sizes = WeatherDatasetWithGraph.load_graph(
+            graph_dir_path=graph_dir_path
+        )
```

Comment:
This does not work as WeatherDatasetWithGraph has no load_graph function. The error passed our cli test as it is currently very limited. It should probably be similar to the code in `test_training.py`:

```suggestion
        graph_features_and_edges = load_graph(graph_dir_path)
        graph_sizes = build_graph_sizes(graph_features_and_edges)
```